svg.go

```diff
@@ -212,11 +213,11 @@ func (s *Svg) UnmarshalXML(decoder *xml.Decoder, start xml.StartElement) error {
 
 			switch tok.Name.Local {
 			case "g":
-				g := &Group{Owner: s, Transform: mt.NewTransform()}
```

Comment:
I would prefer not to change this. I don't think there is any benefit to this change.